src/styles/_catalog.scss

```
@@ -1,22 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
   max-width: 944px;
   margin: 0 auto;
+  gap: 48px;
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 488px) and (max-width: 767px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, 200px);
```

don't repeat 1 column
```suggestion
    grid-template-columns: 200px;
```